### ISSUE TITLE
[A11Y] Proxy buttons misplaced if chart rendered when browser scrolled vertically (draft)

### ISSUE THREAD

**Issue by Necxes** [highcharts/highcharts#23378]
Reopens [17642](https://github.com/highcharts/highcharts/issues/17642) since it is not resolved yet.

#### Expected behaviour
When loading Highcharts on demand with an intersection observer, the `highcharts-a11y-proxy-element` are getting the placed with the same coordinates as their corresponding elements from the legend.

#### Actual behaviour
When loading Highcharts on demand with an intersection observer, the `highcharts-a11y-proxy-element` are getting the placed with the different coordinates as their corresponding elements from the legend.

#### Live demo with steps to reproduce
will follow

#### Product version
Highcharts v12.0.0

#### Affected browser(s)
- Firefox
- Chrome
- Chromium

**Comment by MarkusBarstad** [comment]
I cannot reproduce this. 

Could you supply a demo of this behavior please, with steps to see this behavior?

**Comment by oysteinmoseng** [comment]
Closing as not reproducible for now
